65black/StackOverflow-lite
# #160160178 Add Logic For Get All Answers By User

#### What does this PR do?
This allows a user to get all his/her answers ever posted on StackOverflow-lite.

#### Description of Task to be completed
Have the following endpoint working:
```
GET /api/v1/answers
```
#### How should this be manually tested?
After cloning the repo and installing all dependencies with `npm install`

- Using Postman, sign in and create a new question and a new answer
- Then send a GET request to the above endpoint with your token in your header
- You should get a response with an array of all your previously posted answers

#### Screenshots
![answers](https://user-images.githubusercontent.com/36507972/44854198-d8903480-ac5f-11e8-9ac1-a1623d288653.PNG)

#### What are the relevant pivotal tracker stories?
#160160178
